fab-clientlib/test_containers_fab_rest_api.py

```diff
@@ -0,0 +1,200 @@
+    ]
+
+    token = backend_application_flow(
+        client_id='fab-client-credentials',
+        client_secret='top-secret',
+        token_url='http://localhost:8081/realms/flatland/protocol/openid-connect/token',
+    )
+    print(token)
+    fab = DefaultApi(ApiClient(configuration=Configuration(host="http://localhost:8000", access_token=token["access_token"])))
+    posted_submission = fab.submissions_post(SubmissionsPostRequest(
+        name="fancy",
+        benchmark_definition_id=benchmark_id,
+        submission_data_url="ghcr.io/flatland-association/flatland-benchmarks-f3-starterkit:latest",
+        code_repository="https://github.com/you-name-it",
+        test_definition_ids=[test_id],  # TODO mandatory despite optional in swagger.json, use UUIDs
+        # https://github.com/OpenAPITools/openapi-generator/issues/19485
+        # https://github.com/openAPITools/openapi-generator-pip
+    ))
+    print(posted_submission)
+    submission_id = posted_submission.body.id
+    submissions = fab.submissions_uuid_get(uuid=submission_id)
+    print(submissions)
+    assert submissions.body[0].id == submission_id
+    assert submissions.body[0].benchmark_definition_id == benchmark_id
+    assert submissions.body[0].submitted_by_username == "service-account-fab-client-credentials"
+
+    fab.results_submissions_submission_id_tests_test_id_post(
+        submission_id=submission_id,
+        test_id=test_id,
+        results_submissions_submission_id_tests_test_id_post_request=ResultsSubmissionsSubmissionIdTestsTestIdPostRequest(
+            data=[ResultsSubmissionsSubmissionIdTestsTestIdPostRequestDataInner(
+                scenario_id=scenario_id,
+                additional_properties={key: value}
+            ) for scenario_id, test_id, submission_id, key, value in results]
+        )
+    )
+
+    test_results = fab.results_submissions_submission_id_tests_test_id_get(
+        submission_id=submission_id,
+        test_id=test_id)
+    print("results_uploaded")
+    print(test_results.body)
+    assert test_results.body.scenario_scorings[0].scorings["primary"]["score"] == 100
+    assert test_results.body.scenario_scorings[0].scorings["secondary"]["score"] == 1.0
+    assert test_results.body.scenario_scorings[1].scorings["primary"]["score"] == 101
+    assert test_results.body.scenario_scorings[1].scorings["secondary"]["score"] == 0.8
+    assert test_results.body.scorings["primary"]["score"] == 201
+    assert test_results.body.scorings["secondary"]["score"] == 1.8
+
+    submission_results = fab.results_submissions_submission_id_get(
+        submission_id=submission_id,
+    )
+    assert len(submission_results.body) == 1
+    assert submission_results.body[0].submission_id == submission_id
+    assert len(submission_results.body[0].scorings) == 2
+    assert submission_results.body[0].scorings["primary"]["score"] == 201
```

Comment:
@Holzchopf The `.scorings` and `["score"]` path elements seem artificial - is there a technical reason to have them?